app/javascript/controllers/content_type_controller.js

```diff
@@ -0,0 +1,34 @@
+import { Controller } from 'stimulus'
+
+export default class extends Controller {
+    static targets = [ "direction", "directionRow", "contentType"]
+
+    connect() {
+      this.render()
+    }
+
+    render() {
+      switch (this.currentType()) {
+        case 'https://cocina.sul.stanford.edu/models/image':
+        case 'https://cocina.sul.stanford.edu/models/book':
```

Comment:
image types can have a reading direction set too?

Reply:
Yes, that's exactly the change requested in the ticket.

Reply:
I was surprised but we found it in existing data and the viewer was parsing it correctly.